assets/css/base.css

```diff
@@ -10,7 +10,7 @@ html {
 }
 body{
     font: 14px/24px "Microsoft YaHei","Arial",sans-serif;
-    background:url(/assets/images/beige_paper.png) left top repeat;
+    background:url(../../assets/images/beige_paper.png) left top repeat;

```

Comment:
Can you explain more about all these changes?
I agree with you that relative path in PHP is a little buggy.
But this is HTTP and it handles paths from root directory really well.
I don't think we need these changes.